src/Property/Import.php

```diff
@@ -19,7 +19,7 @@ class Import implements AtRule
     private $location;
 
     /**
-     * @var string
+     * @var string|null
```

Comment:
Could this be `?string`?

Reply:
No, the `?string` notation only works for native type declarations, but not for PHPDoc annotations.